aries-site/src/pages/tokens/overview.mdx

```diff
@@ -0,0 +1,67 @@
+  </Box>
+</Example>
+
+## What is a design token?
+
+Design tokens codify the design decisions that make up a design system's visual style. Tokens replace hard-coded values, such as hex codes for color or px values for spacing, with clear names.
+
+Tokens can represent values including but not limited to colors, typography, spacing, and motion.
+
+An HPE Design Token consists of two parts:
+
+1. A self-explanatory name that represents the coded value: “hpe.color.brand".
+1. An associated value: #01A982.
+
+TO DO: Add visual.
+
+## Why are design tokens important?
+
+Design tokens establish a source of truth for HPE's visual style in a tech-agnostic format, enabling a truly consistent, unified experience across applications.
+
+Token names are purpose-driven, rather than hard-coded values, which provides the flexibility for token values to evolve over time with our visual style. When tokens are implemented across design and development, style updates will propagate consistently across applications simply by pulling in the latest version of the tokens.
+
+Design tokens allow us to:
+
+- Codify design decisions
+- Enable teams across various tech stacks to build efficiently and consistenly with the HPE Design System
+- Sychronize style updates across design and development tooling
+- Build a shared language across design and development
+
+## Types of tokens
+
+Our tokens can be grouped into 3 categories: primitive, semantic, and component. The specificity of design tokens increases from primitive to semantic to component.
+
+### Primitive tokens
+
+Primitive tokens define the most foundational level of the token system, establishing key/value pairs for fundamental attributes like color and dimension.
+
+The value of a primitive token will always be a CSS value such as a hex code, px, or rem value.
+
+Pimitive design tokens should not be used directly. Instead, they are referenced in semantic and component design tokens.
+
+TO DO: Add visual.
+
+### Semantic tokens
+
+Semantic tokens apply a specific meaning to a primitive token to establish how and when a style should be applied. Semantic tokens capture design decisions for color, spacing, elevation, typography, and more.
+
+The value of a semantic token can either be a reference to a primitive token or another semantic token.
+
+Semantic tokens can have different values in different contexts. For example, the value of `hpe.color.background.back` might differ for light vs dark mode. Similarly, the value of `hpe.spacing.large` might differ for wide vs narrow viewports. By building with tokens, the values can flex based on the context.
+
+TO DO: Add visual.
+
+### Component tokens
+
+Component tokens capture specific design decisions for an individual component (such as button) or family of related components (such as inputs).
+
+The value of a component token can either be a reference to a primitive token, semantic token, or another component token.
+
+TO DO: Add visual.
```

Comment:
What do you think about having a brief summary of the token properties. Something like this:

**Design token properties:**

A design token has several properties:

Name (required): name of the token, named according to whether it’s a primitive, semantic or component token. See [How to read tokens](https://deploy-preview-3967--keen-mayer-a86c8b.netlify.app/tokens/how-to-read-tokens).
Value (required): a specific value, for example a hex-code or pixel value for the design token.
Type (optional): The category a token belongs to, for example, color, spacing or corner radius. The type is useful so a tool can properly interpret the type of token it is.
Description (optional): Optional text which is useful for describing how the token should be used.

Reply:
I'm open to adding that since it's listed by the official spec: https://design-tokens.github.io/community-group/format/#design-token-properties

Is it worth adding this to the "What is a design token" section?